packages/bddeb

```diff
@@ -21,6 +22,7 @@ def find_root():
     raise OSError(("Unable to determine where your cloud-init topdir is."
                    " set CLOUD_INIT_TOP_D?"))
 
+
```

Comment:
Drop

Reply:
That's actually a pylint drive-by fix, it requires 2 blank lines there. Are you for dropping as it's unrelated??

Reply:
Well, dropped. I'll fix linting in a separate PR.